src/canonmn_int.cpp

```diff
@@ -741,369 +763,458 @@ namespace Exiv2 {
 
     //! LensType, tag 0x0016
     extern const TagDetails canonCsLensType[] = {
```

Comment:
This is a terrible diff, I'd like this fixed on master to be honest.

Reply:
Fixed in which sense? I understood that the idea is to align these details with what it can be found in **exiftool**.

Reply:
Open the diff on github and enable the switch "hide whitespace changes", then you'll see what I mean: the diff looks like every line got changed, but actually only a subset got touched.

Reply:
Oh I see, I did not notice without that option (I thought that most of the lines changed). 

I understand your point of view, and I would do the work to keep the number of changes to a minimum if it it would be only few lines to be changed. But there are actually few dozens of lines changing here, and actually in the original code the formatting was not nice (note that in lines 745, 747, 750, etc. The first field was not properly aligned with the other lines). 

I would merge the changes as they are (also to have the same version than in 0.27-maintanance). I am not going to spend time on this, but if it is important for you, and you want to deal with this, I will nor merge this.

Reply:
I'll give this a try if I can fix it.

Reply:
whitespace changes are gone now